treeherder/workers/task.py

```diff
@@ -23,6 +23,7 @@ class retryable_task(object):
         jsonschema.ValidationError,
         # eg during log decompression
         zlib.error,
+        KeyError,
```

Comment:
Could you move this up with the other built-in exceptions? (Or else perhaps we should just sort this alphabetically?)

Reply:
Sure thing, I've sorted (case-insensitively) and rebased into this commit.